docs/about/index.rst

```diff
@@ -27,6 +27,13 @@ Project roadmap
 Governance
 ----------
 
+Migrating from Previous Versions
```

Comment:
This paragraph belongs into Quickstart. Probably somewhere at the very top where we talk about installation: https://fairlearn.github.io/master/quickstart.html#installation

Reply:
Done